widgets/image/image.php

```diff
@@ -102,18 +102,22 @@ function get_style_hash($instance) {
 	}
 
 	public function get_template_variables( $instance, $args ) {
+
+		$existing_alt = get_post_meta( $instance['image'], '_wp_attachment_image_alt', true );
```

Comment:
This variable isn't strictly necessary. You could just include the `get_post_meta` call in line 114.